src/frontend/js/components/CourseGlimpse/index.tsx

```diff
@@ -66,17 +67,13 @@ const CourseGlimpseBase = ({ context, course }: CourseGlimpseProps & CommonDataP
           </div>
         ) : null}
         <div className="course-glimpse__metadata course-glimpse__metadata--organization">
-          <svg aria-hidden={true} role="img" className="icon">
-            <use xlinkHref="#icon-org" />
-          </svg>
+          <Icon name="icon-org" title="prout" />
```

Comment:
There is some gas in the code 😅
```suggestion
          <Icon name="icon-org" />
```

Reply:
Well, my huge testing skills are revealed to the public by mistake